### PR TITLE
Related to #467. Adds direct cast to properly call the overloaded member

### PR DESCRIPTION
**Pull request by tomaszstrejczek**
https://github.com/Microsoft/testfx/issues/467

### CODE REVIEW

File: src/Adapter/MSTest.CoreAdapter/Helpers/ReflectHelper.cs [hunk 1]
```diff
@@ -72,7 +72,8 @@ public virtual bool IsAttributeDefined(MemberInfo memberInfo, Type attributeType
         /// <returns>True if the specified attribute is defined on the type.</returns>
         public virtual bool IsAttributeDefined(Type type, Type attributeType, bool inherit)
         {
-            return this.IsAttributeDefined(type.GetTypeInfo(), attributeType, inherit);
+            var mi = (MemberInfo)type.GetTypeInfo();
```
Comment (jayaranigarg) [inline src/Adapter/MSTest.CoreAdapter/Helpers/ReflectHelper.cs]: Consider renaming the variable to memberInfo

**Comment by jayaranigarg** [comment]
@dotnet-bot Test Windows / Debug Build please

File: src/Adapter/MSTest.CoreAdapter/Helpers/ReflectHelper.cs (hunk 1 above)
Comment (tomaszstrejczek) [reply src/Adapter/MSTest.CoreAdapter/Helpers/ReflectHelper.cs]: Renamed

**Comment by jayaranigarg** [comment]
@tomaszstrejczek : Thank you for the contribution 👍 🎉

**Pull request merged**